packages/bindgen/src/realm_js_helpers.h

```diff
@@ -228,6 +230,26 @@ struct Helpers {
     static bool file_exists(const StringData& path) {
         return realm::util::File::exists(path);
     }
+
+    static bool erase_subscription(sync::MutableSubscriptionSet& subs, const sync::Subscription& sub_to_remove) {
```

Comment:
We can't use the by-name and by-query ones for the `removeSubscription()` method, but having an  `eraseById()` seems like the better way to go (replacing the helper `erase_subscription()`). Although, since Core didn't have that available, we created a helper for the time being, utilizing `iterator erase(iterator it)`.